scripts/publish.sh

```diff
@@ -30,5 +45,12 @@ rm public/documentation/index.xml
 rm public/community/index.xml
 rm public/development/index.xml
 
+DESTINATION_STABLE_VERSION="public/documentation/stable"
+echo "Copying the stable documentation from ${STABLE_VERSION_FOLDER} to ${DESTINATION_STABLE_VERSION}"
+cp -r ${STABLE_VERSION_FOLDER} ${DESTINATION_STABLE_VERSION}
+
+echo "Add noindex meta tag to all HTML files in public/documentation/[0-9]*"
+find public/documentation/ -type f -regex "public/documentation/[0-9].*" -name "*.html" | xargs  sed -i '/<\/head>/i\<meta name="robots" content="noindex" \/>'
```

Comment:
Maybe it is something connected with [this SO](http://stackoverflow.com/questions/16745988/sed-command-works-fine-on-ubuntu-but-not-mac). The answer there says to use `sed -i ''`. However, this does not work on my computer, but `sed -i''` works. Can you check with that?